Package.resolved

```diff
@@ -15,7 +15,7 @@
         "repositoryURL": "https://github.com/apple/swift-cmark.git",
         "state": {
           "branch": "gfm",
-          "revision": "5bd1108552a4f54d421d3d39f696952495a4b0cf",
+          "revision": "792c1c3326327515ce9bf64c44196b7f4daab9a6",
```

Comment:
We should wait for https://github.com/apple/swift-markdown/pull/35 to land (and re-run `swift package update` before merging this).